src/overlays/actors/ovl_En_St/z_en_st.c

```diff
@@ -693,7 +693,7 @@ s32 func_808A6A3C(EnSt* this) {
 void func_808A6A78(EnSt* this, PlayState* play) {
     s32 pad[2];
 
-    if (Object_IsLoaded(&play->objectCtx, this->unk_2C0)) {
+    if (Object_IsLoaded(&play->objectCtx, this->objectSlot)) {
```

Comment:
Very odd, when objectSlot is supposed to be GamePlayKeep... I wonder if they moved those assets into gameplay keep when they previously werent?